Makefile

```diff
@@ -364,6 +373,10 @@ ifneq ($(BUILD),local)
 	@echo "$@ docker container is ready to use"
 endif
 
+ifeq ($(findstring adam,$(HV)),adam)
```

Comment:
I am not madly in love with changing the `HV` (hypervisor) to be `adam-kvm`. It isn't the hypervisor we are changing; we are just adding another container to the build.

I think it would be cleaner if we had a separate option, e.g.

```console
make live HV=kvm ADAM=true
```

Reply:
Well, I'm actually about to rename HV into FLAVOR through this experience. These are basically "flavors" of the EVE images that one can build. And sure -- different hypervisors give you different flavors, but so can other things.

What do you think?

Reply:
If we name it FLAVOR (or FLAVOUR :-) ) that is a better reflection, but is FLAVOR a combination of one or more variants? Choosing my hypervisor (one of several enum) and choosing to enable embedded adam (bool) are two distinct knobs.

Suggestion: keep HV and add switch ADAM (`true` or `y` for enable; empty for not). If you also want to make a convenience `FLAVOR` that can include several, then do so. It should be a `-` or `,` or ` ` separated list in any order, and we have some logic that pulls it apart.

E.g. several equivalent ones:

```
make HV=kvm ADAM=true live
make FLAVOR=kvm,adam live
make FLAVOR=adam,kvm live
```

If you prefer to lock it down with the separator as `-`

```
make HV=kvm ADAM=true live
make FLAVOR=kvm-adam live
make FLAVOR=adam-kvm live
```

Reply:
Honestly, given the level of dependencies between adam and kvm it is self-contained flavor at this point (you can't have adam without kvm other). So... I'll just leave it be -- but once we untangle the two -- I'll take your advice.